app/constants.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  REQUEST_USER_FEED: 'Request user feed',
```

Comment:
instead of exporting a huge object as the default export, export them as individual constants:

```js
export const REQUEST_USER_FEED = 'Request user feed';
export const REQUEST_CURRENT_USER = ' Request current user';
```

Then in code you can import like:

```js
import {REQUEST_USER_FEED} from '../constants';
```